ide/static/js/content.js

```diff
@@ -366,15 +366,14 @@ class Content extends React.Component {
   exportPrep(callback) {
     this.dismissAllErrors();
     const error = [];
-    const netObj = JSON.parse(JSON.stringify(this.state.net));
-
-    if (Object.keys(netObj).length == 0) {
+    const net_Obj = JSON.parse(JSON.stringify(this.state.net));
```

Comment:
Minor nitpick - please choose either `camelCase` or `snake_case` and keep it consistent.